src/components/parameter/ParameterEdit.vue

```diff
@@ -124,70 +188,133 @@
 
             <template v-if="state.options.input === 'tickSlider'">
               <v-subheader class="paramLabel" v-text="label()" />
-              <v-tooltip left v-model="state.showResolutionWarning">
-                <template v-slot:activator="{ on, attrs }">
-                  <v-slider
-                    :max="state.options.ticks.length - 1"
-                    :thumb-color="state.color"
-                    :tick-labels="state.options.ticks"
-                    @change="paramChange"
+              <v-slider
+                :hide-details="state.message.length === 0"
+                :hint="state.message"
+                :max="state.options.ticks.length - 1"
+                :persistent-hint="state.message.length > 0"
+                :rules="rules"
+                :thumb-color="state.color"
+                :tick-labels="state.options.ticks"
+                :value="state.value"
+                @change="paramChange"
+                class="mb-1"
+                dense
+                height="40"
+                tick-size="4"
+                ticks="always"
+              >
+                <template #message>
+                  <div
+                    @click="closeMessage"
                     class="mb-1"
```

Comment:
For better readability, I would prefer some spacing on top, together with the changes mentioned in the following lines.
```suggestion
                    class="mb-1 mt-2"